sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/workspace.py

```diff
@@ -41,7 +40,7 @@ class WorkspaceSchema(PathAwareSchema):
     identity = NestedField(IdentitySchema)
     primary_user_assigned_identity = fields.Str()
     workspace_hub = fields.Str(validate=validate_arm_str)
-    managed_network = ExperimentalField(NestedField(ManagedNetworkSchema, unknown=EXCLUDE))
+    managed_network = NestedField(ManagedNetworkSchema, unknown=EXCLUDE)
     enable_data_isolation = fields.Bool()
```

Comment:
Do we need the datastore auth mode here? or that will be something RP team need to work on?

Reply:
created PR for that https://github.com/Azure/azure-sdk-for-python/pull/37072